WebRoot/css/zm.css

```diff
@@ -6585,3 +6579,19 @@ a.ZmLinkDisabled {
 	@MailAttachmentsAllText@
 }
 
+/* Create Contact Group dialog */
+#CreateContactGroupDialog {
+	@CreateContactGroupDialog@
+}
+
+#CreateContactGroupDialog table.ZPropertySheet {
+	@CreateContactGroupDialogTable@
+}
+
+#CreateContactGroupDialog table.ZPropertySheet td {
+	@CreateContactGroupDialogCell@
+}
+
+#CreateContactGroupDialog table.ZPropertySheet td input[type='text'] {
```

Comment:
I don't think we should target that, here we are targeting all text inputs in dialog. Though we have only one now. So if some more new fields are added, it will target that well. 

Let me know if you have any questions.

Thanks